README.md

```diff
@@ -63,31 +63,130 @@ Swagger spec can be found [here](https://hmcts.github.io/reform-api-docs/swagger
 
 ## Onboarding new services
 
-Services will have to have secrets configured with [service-auth-provider-app](https://github.com/hmcts/service-auth-provider-app).
-Once microservice is successfully configured it can be included in [application config](src/main/resources/application.yaml).
-There are 2 places: reports and ftp configuration
+Before a service can be onboarded, it needs to be set up in [service-auth-provider-app (S2S)](https://github.com/hmcts/service-auth-provider-app)
+(see its [README](https://github.com/hmcts/service-auth-provider-app#configuration)).
+Once microservice becomes known to S2S, it can be set up in Send Letter Service's [application config](src/main/resources/application.yaml).
 
-### Reports
+### Creating a new folder on the FTP server
+
+Each request to send a letter results in Send Letter Service uploading a zip file to a service-specific folder on the
+FTP server that belongs to an external provider which prints and posts physical letters.
+In order to enable Send Letter Service to work with a new service, an appropriate folder needs to be created
+on the FTP server (both production and test instance). The change needs to be done by the service provider.
+
+### Changes in application.yaml
+
+This section describes the changes that need to be made in [application.yaml](src/main/resources/application.yaml) file.
+
+#### Reports section
+
+Under `reports.service-config` add a new entry that will provide a displayable name for the service.
+This is needed for creating reports.
 
 ```yaml
 reports:
   service-config:
-    - service: send_letter_tests
-      display-name: Bulk Print # what name to display in the report file
+    - service: ...
+      ...
+    - service: {name of the service in S2S, e.g. divorce}
+      display-name: {displayable name of the service, e.g. Divorce}
 ```
 
-### FTP
+#### FTP section
+
+Under `ftp.service-folders` add a new entry that will provide information about the FTP folder where letters from
+the service should be uploaded. Each service should have its own folder assigned.
 
 ```yaml
 ftp:
   service-folders:
-  - service: send_letter_tests
-    folder: BULKPRINT
+    - service: ...
+      ...
+    - service: {name of the service in S2S, e.g. divorce}
+      folder: {destination folder name, e.g. DIVORCE}
 ```
 
 **Note!**
 This can only be deployed once ftp provider has created folder in all relevant environments.
 
+## Using the service
+
+### Sending a letter request
+
+In order to request sending a letter, retrieve a token from [S2S](https://github.com/hmcts/service-auth-provider-app)
+and call the send letter endpoint. Here's an example `curl` command for this:
+
+```
+curl -X POST -H "Content-Type: application/vnd.uk.gov.hmcts.letter-service.in.letter.v3+json" -H "ServiceAuthorization: Bearer {s2s_token}" "http://rpe-send-letter-service-{env}.service.core-compute-{env}.internal/letters" --data @body.json
+```
```

Comment:
we could just add a link to swagger spec

Reply:
Swagger spec is mentioned in this README. I can also include the link in this section, but it doesn't provide sufficient information for services to be able to talk to send letter.

Reply:
I've added a link to swagger spec.